### ISSUE TITLE
coef.names not set by matrixreg

### ISSUE THREAD

**Issue by hughjonesd** [leifeld/texreg#87]
This is an internal issue, but it looks as if it may cause problems for `texreg` and friends. Using latest master.

In `texreg.R`, matrixreg can be called with `include.attributes` and this is supposed to return attributes including `"gof.names"` and `"coef.names"`. In fact, AFAICS, `"coef.names"` is always `NULL`. This is because `outputmatrix()` is not setting row names on the returned matrix.

Example:
```
library(nlme)
model.1 <- lme(distance ~ age, data = Orthodont, random = ~ 1)
model.2 <- lme(distance ~ age + Sex, data = Orthodont, random = ~ 1)
matrixreg(list(model.1, model.2), include.attributes = TRUE)
```

**Comment by czucca** [comment]
Hi there, 

to be honest I'm not understanding why are you calling matrixreg and not texrex/htmlreg/screenreg. Anyway include.attribute is not an argument specified in matrixreg. I reproduced your code and with  texrex/htmlreg/screenreg it works fine. 

Best, 
C.

**Comment by hughjonesd** [comment]
See my pull request and you’ll understand why this matters :-)

**Comment by czucca** [comment]
I see your point now, but the all idea of matrixreg is to provide a matrix with output that users can customise as they please (see issue #72 and pull request #44 --discussion). Hence the absence of row names is intentional. 

C.

**Comment by hughjonesd** [comment]
I understand. But I assume that the `attr("coef.names")`, which is used internally, is expected to be set? Perhaps it is just redundant code.

What I really need is a way of determining what rows correspond to coefficient names. Is there a way to do that?

**Comment by vincentarelbundock** [comment]
@hughjonesd maybe add a couple new boolean attributes: row.coef row.gof.

**Comment by leifeld** [comment]
Fixed by #118.
